dynamixel_handler/DynamixelCommandPControlCurrent p_cur
dynamixel_handler/DynamixelCommandPControlVelocity p_vel
dynamixel_handler/DynamixelCommandPControlPosition p_pos
dynamixel_handler/DynamixelCommandPControlExtendedPosition p_ext_pos

dynamixel_handler/DynamixelCommandXControlCurrent x_cur
dynamixel_handler/DynamixelCommandXControlVelocity x_vel
dynamixel_handler/DynamixelCommandXControlPosition x_pos
dynamixel_handler/DynamixelCommandXControlExtendedPosition x_ext_pos
dynamixel_handler/DynamixelCommandXControlCurrentPosition x_cur_pos

string    command # "clear_error", "torque_on", "torque_off", "remove_id", "reboot",  "enable", "disable" 
uint16[]  id_list
